apps/quickstart-workdocs/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "monday-workdocs-quickstart-app",
+  "version": "0.0.1",
+  "private": true,
+  "scripts": {
+    "start": "npm run stop && concurrently \"npm run server\" \"npm run expose\"",
+    "build": "react-scripts build",
+    "expose": "mtunnel -p 8306",
```

Comment:
there are repeating issues with our tunnel solution. I suggest removing it altogether from our samples and documentation and suggest ngrok. @arutkowski00 FYI

Reply:
Good call. Will remove the script for now but keep it in the dependencies as an alternative to ngrok.

Reply:
I also added an "Install ngrok" step to the article: 
![image](https://user-images.githubusercontent.com/22015308/218782233-2808b238-de0e-4ec1-b29d-f61505dff81f.png)